sleap_nn/inference/inference.py

```diff
@@ -431,26 +631,45 @@ class TopDownPredictor(Predictor):
 
     """
 
-    centroid_config: Optional[DictConfig] = attr.ib(default=None)
-    confmap_config: Optional[DictConfig] = attr.ib(default=None)
-    centroid_model: Optional[L.LightningModule] = attr.ib(default=None)
-    confmap_model: Optional[L.LightningModule] = attr.ib(default=None)
+    centroid_config: Optional[DictConfig] = attrs.field(default=None)
+    confmap_config: Optional[DictConfig] = attrs.field(default=None)
+    centroid_model: Optional[L.LightningModule] = attrs.field(default=None)
+    confmap_model: Optional[L.LightningModule] = attrs.field(default=None)
 
     def _initialize_inference_model(self):
         """Initialize the inference model from the trained models and configuration."""
         self.model_config = OmegaConf.create()
 
         # Create an instance of CentroidLayer if centroid_config is not None
+        return_crops = False
         if self.centroid_config is None:
             centroid_crop_layer = None
         else:
             self.model_config["centroid"] = self.centroid_config
             self.model_config["data"] = self.centroid_config.inference_config.data
             self.model_config["skeletons"] = self.centroid_config.data_config.skeletons
+            if self.confmap_model:
+                return_crops = True
+            labels = sio.load_slp(
+                self.centroid_config.inference_config.data.labels_path
+            )
```

Comment:
Definitely not. We should 100% not load an SLP just so we can pull out a single scalar parameter!! Store it in the model config!